### PR TITLE
Ethan refactor views

### LINKED ISSUE #50: Refactor Views into Folders

### PR DESCRIPTION
**Pull request by EVanGorkom**
## Implements/Fixes: Refactor Views into Separate Files

  This PR will refactor all of the logic for each model into its own separate `views` file as well as refactor the urls to reach these new files accordingly. 

## This closes issue:
 Closes #50 

## Necessary checkmarks:
- [x] All Tests are Passing
- [x] The code will run locally

## Type of change
- [ ] New feature
- [ ] Bug Fix

## Check the correct boxes
- [x] This broke nothing
- [ ] This broke some stuff
- [ ] This broke everything

## Testing Changes
- [x] No Tests have been changed
- [ ] Some Tests have been changed
- [ ] All of the Tests have been changed(Please describe what in the world happened)

## Checklist:
- [x] My code has no unused/commented out code
- [x] I have reviewed my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have fully tested my code

TY!

### CODE REVIEW

File: market_nextdoor_api/urls.py [hunk 1]
```diff
@@ -16,24 +16,24 @@
 """
 from django.contrib import admin
 from django.urls import path
-from market_nextdoor_api import views
+from .views import market_views, customer_views, vendor_views, item_views, customer_views, preorder_views
 from rest_framework.urlpatterns import format_suffix_patterns
```
Comment (kbug819) [inline market_nextdoor_api/urls.py:20]: Great implementation of what Antoine mentioned. This looks great, and glad to know we can import so many things this way!

**Review by kbug819 — APPROVED** [review]
Looks great, thanks for doing so much on this refactor!

File: market_nextdoor_api/urls.py [hunk 2]
```diff
@@ -16,24 +16,24 @@
 """
 from django.contrib import admin
 from django.urls import path
-from market_nextdoor_api import views
+from .views import market_views, customer_views, vendor_views, item_views, customer_views, preorder_views
```
Comment (Antoine-Aube) [inline market_nextdoor_api/urls.py:19]: yeah this is pretty much exactly what I had seen my research!

**Comment by Antoine-Aube** [comment]
Yeah I wish I had more to put on here as far as review but this looks like what we had chatted about. I think there's a couple ways to do it, but this one feels like the most appropriate for the scale of this project. Thanks for taking on this cleaning task Ethan!

**Review by Antoine-Aube — APPROVED** [review]

**Pull request merged**